src/net.h

```diff
@@ -817,7 +817,7 @@ class CNode
     std::atomic_bool fPauseSend{false};
 
 protected:
-    mapMsgCmdSize mapSendBytesPerMsgCmd;
+    mapMsgCmdSize mapSendBytesPerMsgCmd GUARDED_BY(cs_vSend);
```

Comment:
9e92f63e2deadbfd7424ce4585f02ad580dcee8c

Is there any reason to keep `mapSendBytesPerMsgCmd` `protected`?

Reply:
Yes, it's to keep it consistent with `nSendBytes` for calls to the `getpeerinfo` rpc.

Reply:
There's no reason for either of these to be protected, since nothing inherits from `CNode`. However, that's separate from this PR and could be fixed elsewhere.